shared/src/commonTest/kotlin/co/touchlab/kampstarter/BreedModelTest.kt

```diff
@@ -7,13 +7,18 @@ import co.touchlab.kampstarter.response.BreedResult
 import co.touchlab.kermit.Kermit
 import com.russhwolf.settings.MockSettings
 import kotlinx.coroutines.CompletableDeferred
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.flow.collect
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.flowOn
 import kotlin.test.*
 
 class BreedModelTest: BaseTest() {
 
     private lateinit var model: BreedModel
     private val kermit = Kermit()
-    private var dbHelper = DatabaseHelper(testDbConnection(), kermit)
+    private var dbHelper = DatabaseHelper(testDbConnection(), kermit, Dispatchers.Main)
```

Comment:
Should we be passing `Dispatchers.Default` here as well? The tests previously would have run on that dispatcher since it wasn't injectable.

Reply:
Right, using Default now